.env

```diff
@@ -0,0 +1 @@
+GENERATE_SOURCEMAP=false
```

Comment:
does it mean local server and/or test stands won't have source maps?

Reply:
This is set for production. 

From docs:

When set to false, source maps are not generated for a production build. This solves out of memory (OOM) issues on some smaller machines.

https://create-react-app.dev/docs/advanced-configuration/